src/other/Table.js

```diff
@@ -201,5 +228,27 @@
         this._currentSort = column;
         this.render();
     };
+
+    Table.prototype.Selection = function (_) {
+        if (!arguments.length) {
+            var t = [];
+            for (var i = 0; i < this._selection.get().length; i++) {
+                t[i] = this._selection.get()[i]._id;
+            }
+            return t;
+        } else {
+            var t = [];
+            for (var i = 0; i < _.length; i++) {
+                t[i] = {
```

Comment:
i'd do the ind. on this a bit dif:
t[i] = {
	"_id": _[i],
	element: function() {
	      return d3.select(null);
	}
}